<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdicionarConfDlg</class>
 <widget class="QDialog" name="AdicionarConfDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adicionar Configuração Regional</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>12</y>
     <width>484</width>
     <height>494</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Configuração Regional</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLineEdit" name="novaRegiaoLineEdit"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Estados</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QListWidget" name="naoInclusosLista">
            <property name="minimumSize">
             <size>
              <width>51</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>51</width>
              <height>91</height>
             </size>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::MultiSelection</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>23</width>
                <height>38</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="includeToolButton">
              <property name="text">
               <string>...</string>
              </property>
              <property name="arrowType">
               <enum>Qt::RightArrow</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="removeToolButton">
              <property name="text">
               <string>...</string>
              </property>
              <property name="arrowType">
               <enum>Qt::LeftArrow</enum>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_5">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="inclusosLista">
            <property name="minimumSize">
             <size>
              <width>51</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>51</width>
              <height>91</height>
             </size>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::MultiSelection</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_10">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>191</width>
          <height>141</height>
         </size>
        </property>
        <property name="title">
         <string>Reserva Útil de Água</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>152</width>
           <height>94</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Solo Tipo 3</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="soloSpinBox3">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="suffix">
             <string> mm</string>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="maximum">
             <number>300</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="soloSpinBox1">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="suffix">
             <string> mm</string>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="maximum">
             <number>300</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Solo Tipo 2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="soloSpinBox2">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="suffix">
             <string> mm</string>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="maximum">
             <number>300</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Solo Tipo 1</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Tipo Velocidade Radicular</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="tipoVradComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToContents</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>17</width>
            <height>13</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Grupos</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QListWidget" name="gruposLista">
          <property name="maximumSize">
           <size>
            <width>101</width>
            <height>81</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="adicionarGrupoButton">
            <property name="maximumSize">
             <size>
              <width>21</width>
              <height>21</height>
             </size>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removerGrupoButton">
            <property name="maximumSize">
             <size>
              <width>21</width>
              <height>21</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editarGrupoButton">
            <property name="maximumSize">
             <size>
              <width>21</width>
              <height>21</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_11">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>36</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>novaRegiaoLineEdit</tabstop>
  <tabstop>soloSpinBox1</tabstop>
  <tabstop>soloSpinBox2</tabstop>
  <tabstop>soloSpinBox3</tabstop>
  <tabstop>removeToolButton</tabstop>
  <tabstop>inclusosLista</tabstop>
  <tabstop>includeToolButton</tabstop>
  <tabstop>naoInclusosLista</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>tipoVradComboBox</tabstop>
  <tabstop>gruposLista</tabstop>
  <tabstop>adicionarGrupoButton</tabstop>
  <tabstop>removerGrupoButton</tabstop>
  <tabstop>editarGrupoButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AdicionarConfDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>505</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AdicionarConfDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>504</x>
     <y>505</y>
    </hint>
    <hint type="destinationlabel">
     <x>482</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>naoInclusosLista</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>includeToolButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inclusosLista</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>removeToolButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gruposLista</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>editarGrupoButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
